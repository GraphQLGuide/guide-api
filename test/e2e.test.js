import { gql, startE2EServer } from 'guide-test-utils'

let stop, request

beforeAll(async () => {
  const server = await startE2EServer()
  stop = server.stop
  request = server.request
})

afterAll(() => stop())

const HELLO = gql`
  query {
    hello
  }
`

test('hello', async () => {
  const result = await request({ query: HELLO })

  expect(result).toMatchSnapshot()
})
