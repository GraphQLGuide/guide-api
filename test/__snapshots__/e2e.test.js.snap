// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello 1`] = `
Object {
  "data": Object {
    "hello": "🌍🌏🌎",
  },
}
`;
